src/generic_ui/polymer/cloud-install.ts

```diff
@@ -7,28 +7,36 @@ import ui_constants = require('../../interfaces/ui');
 
 var ui = ui_context.ui;
 
-const INSTALL_ARGS: uproxy_core_api.CloudInstallArgs = {
-  providerName: 'digitalocean'
+const DEFAULT_REGION = 'nyc3'
+
+var install_args: uproxy_core_api.CloudInstallArgs = {
```

Comment:
It's kinda weird to keep this variable around. Suggest you remove this, make a `const` for both `DEFAULT_REGION` and `DEFAULT_PROVIDER`, and just pass an object to `ui.cloudInstall` (line 39).

In fact, since the UI will now always pass a region, is it even worth keeping `DEFAULT_REGION`?